config.md

```diff
@@ -94,3 +95,13 @@ service:
     filename: "{title}.s{season}e{episode}.{episodename}-{service}.{ext}"
 ```
 
+This will download all episodes with subtitles and audio merged in to the video, sort them in to subfolders within ~/svtplay-dl and finally write the file name without the service attached.
+```
```

Comment:
- Need to add yaml markup language here

Reply:
This is now fixed thanks to https://github.com/spaam/svtplay-dl-web/commit/fcc2a0be7a0b54f36987b145c9ad6dcc012aa6fe